test/functional/sync_plan/delete_test.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../test_helper'
```

Comment:
You don't need to add tests here. Just for create, since that's what this PR is about.